pulsar-broker/src/test/java/org/apache/pulsar/broker/service/OneWayReplicatorTest.java

```diff
@@ -1160,4 +1160,77 @@ public void testDifferentTopicCreationRule(ReplicationMode replicationMode) thro
         admin1.namespaces().deleteNamespace(ns);
         admin2.namespaces().deleteNamespace(ns);
     }
+
+    @Test(dataProvider = "replicationModes")
+    public void testDisabledCreateTopicToRemoteClusterForReplication(ReplicationMode replicationMode) throws Exception {
```

Comment:
Could you point out where disabled `createTopicToRemoteClusterForReplication` ?

Reply:
This test was added in mistake, I removed this test